src/main/java/hw_aqa_3/task4.java

```diff
@@ -0,0 +1,41 @@
+package hw_aqa_3;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+
+import java.time.Duration;
+
+public class task4 {
```

Comment:
Class name starts from the small letter?

Reply:
fixed